ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -8,113 +8,181 @@
     {
       id: 'VarisEx Phase 2',
       regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CCC', source: 'Varis yae Galvus', capture: false }),
```

Comment:
Sorry for such a small comment, but you can write this as source first, then id, to match with the above, and to match other files?

Reply:
Alright, but I am confused which should be first? In [titania-ex.js](https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/05-shb/trial/titania-ex.js) and [innocense-ex.js](https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/05-shb/trial/innocence-ex.js), there is id-first, but in [ruby-weapon-ex.js](https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js), that id source-first. So I didn't know which is the preferred one.
For this file, I will change it as id-first soon.